docs/sources/http_api/dashboard.md

```diff
@@ -541,3 +541,97 @@ Content-Length: 97
   "status": "multiple-slugs-exists"
 }
 ```
+
+## Trim input dashboard
+
+`POST /api/dashboards/trim`
+
+Will remove the default values from input dashboard JSON.
+
+**Example Request for trimming dashboard JSON**:
+
+```http
+POST /api/dashboards/uid/trim HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+
+{
+    "meta": {
```

Comment:
Is meta really included/required in the request?

Reply:
it is not required, but allowed. would you recommend not put it in the example?